mdx/guides/0x-api-faq.mdx

```diff
@@ -0,0 +1,142 @@
+## Why is there varying `to` addresses provided in the quotes by 0x API?
+
+0x API provides quotes that execute with a few 0x protocol smart contract interfaces. Each interface provides specific user experiences that suite different swap use cases. 0x API selects the best 0x protocol interface to interact with based on the provided parameters and the smart order routing logic. 
+
+Currently, there are two main interfaces that 0x API will provide quotes for:
+
+**Exchange contract** - the main canonical interface for the 0x protocol, all swap quotes are settled by the exchange contract.
+
+**Forwarder extension contract** - a payable interface allowing swaps between native ETH (as a `sellToken`) and another ERC20 asset.
+
+Find the currently deployed contracts [here](https://0x.org/docs/guides/0x-cheat-sheet).
+
+Astute developer will note that the exchange contract functions are `payable`. The `payable` interface is utilized to pay 0x v3’s protocol fee and NOT engineered to allow users to swap tokens with native ETH. Forwarder, however, is designed to meet the native ETH swapping user experience.
+
+## When does forwarder or exchange get used by 0x API/how can I ensure one is used?
+
+The current 0x API logic for selecting between forwarder or exchange is deterministic and enable developers to easily ensure which interface is used for a quote:
+
+* If `sellToken` is not set to `ETH` (say `WETH`), 0x API will always provide a quote that interfaces with the main exchange contract.
+* If `sellToken` is set to `ETH`, 0x API will always provide a quote that interfaces with forwarder contract.
+
+**TLDR** `sellToken=ETH` uses Forwarder, `sellToken=WETH` uses Exchange.
+
+## Are all these to smart contracts audited/safe?
+
+The 0x core team holds itself to a high security standard with all the smart contracts written; 0x protocol’s contracts are open source and extensively tested by 0x core team’s internal protocol team and by external auditors.
+
+The [**Exchange**](https://0x.org/docs/guides/0x-cheat-sheet) contract, because they are the main canonical interface of the 0x protocol, is versioned, and governed by ZRX token holders. Because of its critical place in 0x infrastructure, the 0x core team employs a number of external audits to ensure the safe and intended usage of the 0x contract.
```

Comment:
Again, I would start with: is audited by XYZ firm, and then go into all the additional things we do.